Add Google Nest breaking change to 2023-08 release notes (#28473)
home-assistant/home-assistant.github.io: source/_posts/2023-08-02-release-20238.markdown

{% enddetails %}

{% details "Google Nest" %}

> The legacy Works With Nest API is no longer supported. You must manually take

Reviewer: Maybe just put the reference to the Nest Announcement here. At the bottom it's a bit redundant, isn't it?
```suggestion
Since the Works With [Nest API is shutting down by September 2023](https://developers.nest.com/), we removed the support for it. You must...
```

Author: It's not true though, this was planned first. I don't want to pass the buck to Google, this legacy unmaintained integration needs to die for the assistant specific reasons.  It was deprecated over a year ago and I had to temporarily undo it.

Maybe I'm bitter....

Reviewer: Ok, didn't know that. In that case you can just leave it. 👍

> The legacy Works With Nest API is no longer supported. You must manually take
> action to use the SDM API by removing nest configuration from configuration.yaml
> and restarting Home Assistant, then add the Nest integration following the
> instructions. Please review the documentation for information about which
> devices are supported.

Reviewer 2: ```suggestion
The legacy Works With Nest API is no longer supported. You must manually take
action to use the SDM API by removing the Nest configuration from `configuration.yaml`
and restarting Home Assistant. Then add the Nest integration following the
instructions. Please review the documentation for information about which
devices are supported.
```

> Additionally, as a reminder, the legacy Works With Nest API is [shutting down](https://developers.nest.com/)
> in September 2023.

Reviewer: Remove this:
```suggestion
```

Author: Are you suggestion this because of my other comment about not blaming nest? I think your suggestion of linking to the deprecation docs seemed helpful, so i'm ok to leave this and its verbatim from the breaking change PR.

Reviewer: I don’t know. I think we could add it back in and then merge the PR.

Author: OK, its there now.
